src/index.ts

```diff
@@ -0,0 +1,2 @@
+import Kuery from './Kuery';
+export default Kuery;
```

Comment:
Perhaps we want to export everything from `./Kuery`, to include the `KueryOptions` as well?

```ts
export * from './Kuery';
```

Reply:
Realized they're only used internally. Don't export the interface.